drivers/bridge/bridge_test.go

```diff
@@ -72,8 +72,8 @@ func TestCreateFail(t *testing.T) {
 
 func TestQueryEndpointInfo(t *testing.T) {
 	defer netutils.SetupTestNetNS(t)()
-
-	_, d := New(nil)
+	d := New()
+	dd, _ := d.(*driver)
```

Comment:
Please check on returned error in each call to bridge.New() in the bridge testing code.